Source/Scene/Scene.js

```diff
@@ -2731,7 +2732,16 @@ define([
                 pickedPrimitives.push(primitive);
             }
 
+            prevPickedResult = pickedResult;
             pickedResult = this.pick(windowPosition);
+
+            //>>includeStart('debug', pragmas.debug);
+            if (pickedResult === prevPickedResult) {
+                throw new DeveloperError(
```

Comment:
This would mask a bug elsewhere - basically setting a primitive's `show` to `false` not working.  Can you remove this and keep the scope of this bug to picking the atmosphere?